modules/light-clients/07-tendermint/proposal_handle_test.go

```diff
@@ -65,77 +65,29 @@ func (suite *TendermintTestSuite) TestCheckSubstituteUpdateStateBasic() {
 	}
 }
 
-// to expire clients, time needs to be fast forwarded on both chainA and chainB.
-// this is to prevent headers from failing when attempting to update later.
 func (suite *TendermintTestSuite) TestCheckSubstituteAndUpdateState() {
 	testCases := []struct {
-		name         string
-		FreezeClient bool
-		ExpireClient bool
-		expPass      bool
+		name    string
+		expPass bool
 	}{
 		{
-			name:         "PASS: update checks are deprecated, client is frozen and expired",
-			FreezeClient: true,
-			ExpireClient: true,
-			expPass:      true,
-		},
-		{
-			name:         "PASS: update checks are deprecated, not frozen or expired",
-			FreezeClient: false,
-			ExpireClient: false,
-			expPass:      true,
-		},
-		{
-			name:         "PASS: update checks are deprecated, not frozen or expired",
-			FreezeClient: false,
-			ExpireClient: false,
-			expPass:      true,
-		},
-		{
-			name:         "PASS: update checks are deprecated, client is frozen",
-			FreezeClient: true,
-			ExpireClient: false,
-			expPass:      true,
-		},
-		{
-			name:         "PASS: update checks are deprecated, client is expired",
-			FreezeClient: false,
-			ExpireClient: true,
-			expPass:      true,
+			name:    "PASS: update checks are deprecated",
+			expPass: true,
 		},
 	}
 
 	for _, tc := range testCases {
 		tc := tc
 
-		// for each test case a header used for unexpiring clients and unfreezing
-		// a client are each tested to ensure that unexpiry headers cannot update
-		// a client when a unfreezing header is required.
 		suite.Run(tc.name, func() {
-			// start by testing unexpiring the client
 			suite.SetupTest() // reset
 
 			// construct subject using test case parameters
 			subjectPath := ibctesting.NewPath(suite.chainA, suite.chainB)
 			suite.coordinator.SetupClients(subjectPath)
 			subjectClientState := suite.chainA.GetClientState(subjectPath.EndpointA.ClientID).(*ibctm.ClientState)
 
-			// apply freezing or expiry as determined by the test case
-			if tc.FreezeClient {
-				subjectClientState.FrozenHeight = frozenHeight
-			}
```

Comment:
We should probably leave the freeze client cases in since we unfreeze the client if it is frozen in the code

Reply:
ah, yea good point